components/espcoredump/espcoredump.py

```diff
@@ -1624,22 +1624,25 @@ def gdb2freertos_thread_id(gdb_thread_id):
     p = gdbmi_getinfo(p, handlers, "info threads")
     # THREADS STACKS
     p,threads,cur_thread = gdbmi_get_thread_ids(p)
+    print()
     for thr_id in threads:
         task_index = int(thr_id) - 1
-        if thr_id == cur_thread:
-            continue
```

Comment:
I would leave this, since stack of the current thread is printed at the beginning.

Reply:
Hi Alexey,

I recognized that information about the current thread stack was printed in the beginning. 

However, I found it convenient to have the information complete within the whole "THREADS INFO" section of the output. That way, users do not have to scroll back and forth when analyzing all the stack information.

Does this make sense to you?

With kind regards,
Andreas.

Reply:
Yes, it makes sense